app/styles/input.css

```diff
@@ -85,7 +86,7 @@ textarea {
     /* Make textareas show at minimum one line. This does not work when
        using box-sizing border-box, in which case, vertical padding and
        border width needs to be taken into account. */
-    min-height: 1em;
+    min-height: calc(1em * var(--line-height));
```

Comment:
Have you checked if there are any magical CSS units that might do what we want without `calc()`?

Reply:
Sweet, you are right! There is a unit that does what we want!! `1lh` :fire: :confetti_ball: :rocket:

Updated to use that one